tests/Unit/PointwiseFunctions/Hydro/TestFunctions.py

```diff
@@ -36,7 +36,25 @@ def mass_weighted_coords_b(tilde_d, grid_coords, compute_coords):
     return tilde_d * compute_coords * np.heaviside(grid_coords * (-1.0), 0)
 
 
-# Functions testing the MassWeightedFluidItems
+# End functions for testing the MassWeightedFluidItems
+
+# Functions testing QuadrupoleFormula
+
+
+def quadrupole_moment(tilde_d, compute_coords):
```

Comment:
We've been moving to separate `.py` files because these big ones get a bit unwieldy. Could you move your new functions to `QuadrupoleFormula.py`? It's slow going updating the existing code, that's why this file still exists. Updates like this are just never a high priority